src/cluster/managers/moderation/ChatLogManager.ts

```diff
@@ -52,7 +52,7 @@ export class ChatLogManager {
     }
 
     public async messageUpdated(message: Message<PossiblyUncachedTextableChannel>): Promise<void> {
-        if (!guard.isGuildMessage(message) || await this.cluster.database.guilds.getSetting(message.channel.guild.id, 'makelogs') !== true)
+        if (!guard.isGuildMessage(message) || await this.cluster.database.guilds.getSetting(message.channel.guild.id, 'makelogs') !== true || !guard.hasValue(message.author))
```

Comment:
Why do you add a check on the author? Wouldn't the event be fired only when an author updates it?

Reply:
Yep, I know, and yet
![image](https://user-images.githubusercontent.com/7736591/174474827-521d74d2-d191-4b20-9391-e5841e1b0f71.png)
Eris has some really bad typings :( apparently there is a scenario where a message update has no author